File: app/src/main/java/cm/aptoide/pt/home/RewardApp.java
package cm.aptoide.pt.home;

import cm.aptoide.pt.view.app.Application;

public class RewardApp extends Application {
  private String clickUrl;
  private String downloadUrl;
  private float reward;

  public RewardApp(String appName, String appIcon, float ratingAverage, int downloadsNumber,
      String packageName, long appId, String tag, boolean hasBilling, String clickUrl,
      String downloadUrl, float reward) {
    super(appName, appIcon, ratingAverage, downloadsNumber, packageName, appId, tag, hasBilling);
    this.clickUrl = clickUrl;
    this.downloadUrl = downloadUrl;
    this.reward = reward;
  }

  public String getClickUrl() {
    return clickUrl;
  }

  public String getDownloadUrl() {
    return downloadUrl;
  }

  public float getReward() {
    return reward;
  }
}

File: old_app/src/main/java/cm/aptoide/pt/home/bundles/apps/RewardApp.kt
package cm.aptoide.pt.home.bundles.apps

import cm.aptoide.pt.view.app.Application

class RewardApp(appName: String, appIcon: String, ratingAverage: Float, downloadsNumber: Int,
                packageName: String, appId: Long, tag: String, hasBilling: Boolean,
                val clickUrl: String? = "", val downloadUrl: String? = "", val reward: Reward?,
                val featureGraphic: String? = "") :
    Application(appName, appIcon, ratingAverage, downloadsNumber, packageName, appId, tag,
        hasBilling) {

  data class Reward(val appc: Double, val fiat: Fiat)
  data class Fiat(var amount: Double = -1.0, var currency: String = "", var symbol: String = "")

}